src/routes/cache-hooks/cache-hooks.controller.ts

```diff
@@ -17,7 +18,9 @@ export class CacheHooksController {
   @UseGuards(PreExecutionLogGuard, BasicAuthGuard)
   @Post('/hooks/events')
   @HttpCode(202)
-  async postEvent(@Body(EventValidationPipe) event: Event): Promise<void> {
+  async postEvent(
+    @Body(new ValidationPipe(WebHookSchema)) event: Event,
```

Comment:
What is the response code if the we receive an event that we don't know how to handle?

Reply:
Nevermind, forgot that we had a global exception filter:

https://github.com/safe-global/safe-client-gateway/blob/ba418d0cd54e5074384a8710ff9176ea11f9ca86/src/routes/common/filters/zod-error.filter.ts#L25-L39

So the expected status code would be 422 which I would say is correct for this route 👍